src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
```

Comment:
Since the star and star-active images are related and used together, consider using a sprite sheet for these images to reduce the number of HTTP requests and improve loading performance.